packages/manager/apps/public-cloud/src/index.js

```diff
@@ -59,6 +60,7 @@ angular
     navbar,
     preload,
     walkMe,
+    'ngAnimate',
```

Comment:
According to my previous comment (import dependencies in that way):

```suggestion
    ngAnimate,
```